english/tos/tos.md

```diff
@@ -0,0 +1,3 @@
+[Tree of Savior](https://treeofsavior.com/) is the spiritual successor
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.